media/css/firefox/firstrun/firstrun.scss

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$break-large: 1305px;
+
+@import '../../../protocol/css/includes/lib';
+
+
+//* -------------------------------------------------------------------------- */
+// Common elements & variables
+
+html {
+    height: 100%;
+    min-height: 100vh;
+}
+
+section .mzp-l-content, div.mzp-c-hero-image, .browser-logo img {
+    display: inline-block;
+    max-width: 320px;
+    text-align: left;
+}
+
+.agreement {
+    padding-bottom: $spacing-sm;
+}
+
+.browser-logo img {
+    display: block;
+    max-width: 250px;
+    padding: $spacing-lg 0 $spacing-xl;
+}
+
+.field {
+    margin: 0;
+    padding: 0;
+}
+
+.mzp-c-button {
+    width: 100%;
+}
+
+.mzp-c-hero-desc {
+    padding-bottom: $spacing-md;
+}
+
+.mzp-c-hero-image {
+    margin-top: $spacing-sm;
+}
+
+.mzp-c-hero-title {
+    @include text-display-md;
+}
+
+.fxa-email-field {
+    width: 100%;
+}
+
+.fxa-email-form-intro, .fxa-email-form-intro strong {
+    font-weight: bold;
+    @include text-display-xs;
+    display: inline;
+}
+
+.fxa-email-field-container {
+    margin-top: $spacing-lg;
+}
+
+.fxa-signin {
+    @include text-body-xs;
+    margin-top: $spacing-sm;
+}
+
+.fxa-signin, .agreement {
+    color: $color-gray-70;
+}
+
+.page-content {
+    margin-bottom: $spacing-xl;
+    text-align: center;
+}
+
+/* -------------------------------------------------------------------------- */
+// Emphasis Box style, this should go away when Protocal PR #441 is landed. Issue #385
+
+.c-box-emphasis {
+    background: $color-white;
+    border-radius: 0.2em;
```

Comment:
From the mockup I think these corners should be a little bit rounder. I'm not sure of the exact radius and we haven't really standardized these yet... looks like maybe 6px? Or 8px? We can check the Sketch file.

Reply:
it was 6px, I just figured out how to find it 🤦‍♀.